### PR TITLE
Add TourGuide-App 'Installation' section

### PR DESCRIPTION
**Pull request by albertinisg**

### CODE REVIEW

File: doc/fiware-tour-guide-application-a-tutorial-on-how-to-integrate-the-main-fiware-ges/installation.md [hunk 1]
```diff
@@ -1 +1,51 @@
...
+# Installation
+
+## Installing Docker
+
+First of all you need to install Docker. Docker is a container technology which allows to instantiate different FIWARE components isolated into their respective environments. Below you can find relevant links which describe how to install Docker on different platforms.
+
+### Windows
+
+You have two options for installing Docker on Windows depending on the requirements.
+
```
Comment (jmcanterafonseca) [inline doc/fiware-tour-guide-application-a-tutorial-on-how-to-integrate-the-main-fiware-ges/installation.md]: we can get rid of this sentence

File: doc/fiware-tour-guide-application-a-tutorial-on-how-to-integrate-the-main-fiware-ges/installation.md [hunk 2]
```diff
@@ -1 +1,51 @@
...
+Find detailed information on how to install it [here](https://docs.docker.com/engine/installation/windows/).
+
+### Mac
+
+You have two options for installing Docker on Mac OS X depending on the requirements.
+
+Find detailed information on how to install it [here](https://docs.docker.com/engine/installation/mac/).
```
Comment (jmcanterafonseca) [inline doc/fiware-tour-guide-application-a-tutorial-on-how-to-integrate-the-main-fiware-ges/installation.md]: you can get rid of this sentence

File: doc/fiware-tour-guide-application-a-tutorial-on-how-to-integrate-the-main-fiware-ges/installation.md [hunk 3]
```diff
@@ -1 +1,51 @@
...
+
+### Linux
+
+Docker is supported on the main Linux distributions. Find detailed information on how to install it [here](https://docs.docker.com/engine/installation/linux).
+
+## Installing Docker-compose
+
+Next step is installing docker-compose. This tool allows to instantiate different containers that have dependencies between them. The configuration of the containers to be instantiated is normally described by a file names docker-compose.yml.
```
Comment (jmcanterafonseca) [inline doc/fiware-tour-guide-application-a-tutorial-on-how-to-integrate-the-main-fiware-ges/installation.md]: nit : s/names/named/

File: doc/fiware-tour-guide-application-a-tutorial-on-how-to-integrate-the-main-fiware-ges/installation.md [hunk 4]
```diff
@@ -1 +1,51 @@
...
+By installing Docker (any method) it satisfies this requirement.
+
+### Mac
+
+By installing Docker (any method) it satisfies this requirement.
+
+### Linux
+
+A complete guide for installing Docker Compose can be found here: https://docs.docker.com/compose/install/.
+
```
Comment (jmcanterafonseca) [inline doc/fiware-tour-guide-application-a-tutorial-on-how-to-integrate-the-main-fiware-ges/installation.md]: md link needed

File: doc/fiware-tour-guide-application-a-tutorial-on-how-to-integrate-the-main-fiware-ges/installation.md [hunk 5]
```diff
@@ -1 +1,51 @@
...
+## Clone repository
+
+Once you have all the tools needed you can start by cloning the Github repository of the Tour Guide Application. If you have not previously installed git, please see this [link](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) which explains how to install git.
```
Comment (jmcanterafonseca) [inline doc/fiware-tour-guide-application-a-tutorial-on-how-to-integrate-the-main-fiware-ges/installation.md]: s/see/have a look at/

**Comment by albertinisg** [comment]
Changes addressed in 8ebeb9b37c5b095f8699a9a59ebdc0d97569de5a

File: doc/fiware-tour-guide-application-a-tutorial-on-how-to-integrate-the-main-fiware-ges/installation.md [hunk 6]
```diff
@@ -1 +1,47 @@
-# To be replaced with content
+# Installation
+
+## Installing Docker
+
+First of all you need to install Docker. Docker is a container technology which allows to instantiate different FIWARE components isolated into their respective environments. Below you can find relevant links which describe how to install Docker on different platforms.
+
+### Windows
+
+Find detailed information on how to install it [here](https://docs.docker.com/engine/installation/windows/).
```
Comment (jmcanterafonseca) [inline doc/fiware-tour-guide-application-a-tutorial-on-how-to-integrate-the-main-fiware-ges/installation.md]: You can ...

File: doc/fiware-tour-guide-application-a-tutorial-on-how-to-integrate-the-main-fiware-ges/installation.md [hunk 7]
```diff
@@ -1 +1,47 @@
...
+
+### Mac
+
+Find detailed information on how to install it [here](https://docs.docker.com/engine/installation/mac/).
```
Comment (jmcanterafonseca) [inline doc/fiware-tour-guide-application-a-tutorial-on-how-to-integrate-the-main-fiware-ges/installation.md]: same comment

**Comment by albertinisg** [comment]
Changes amended

**Pull request merged**